content/en/user-guide/tools/multi-account-setups/index.md

```diff
@@ -5,6 +5,7 @@ categories: ["LocalStack Pro"]
 tags: ["multi-tenant", "multi-account", "account-id", "namespaces"]
 aliases:
   - /aws/multi-account-setups/
```

Comment:
do we still need this alias?

Reply:
Possibly yes, since this was part of AWS docs before